src/main/java/org/folio/bulkops/mapper/UserToUserDtoMapperImpl.java

```diff
@@ -0,0 +1,21 @@
+package org.folio.bulkops.mapper;
+
+import java.util.UUID;
+import org.folio.bulkops.domain.bean.Personal;
+import org.folio.bulkops.domain.bean.User;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserToUserDtoMapperImpl implements UserToUserDtoMapper {
```

Comment:
org.folio.bulkops.domain.bean.User - entity for csv-file processing (with whole data), org.folio.bulkops.domain.dto.User - entity with a limited set of fields to show information about user who executed bulk operations. So, this looks like a mix of two user entities from absolutely different business flows. As a result this mapper introduces confusion into the code. Is it possible to get rid of it and transform the fields as was done originally?

Reply:
done